cpp/data/zotero_cgi/qa.html

```diff
@@ -0,0 +1,67 @@
+        <link rel="stylesheet" href="https://cdn.datatables.net/1.10.16/css/jquery.dataTables.min.css"/>
+        <script type="text/javascript" src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+        <script type="text/javascript" src="https://code.jquery.com/ui/1.12.0/jquery-ui.min.js"></script>
+        <script type="text/javascript" src="https://cdn.datatables.net/v/dt/dt-1.10.16/datatables.js"></script>
+
+        <!-- popper is required by bootstrap for tabs -->
+        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js"></script>
+        <script type="text/javascript" src="https://getbootstrap.com/docs/4.0/dist/js/bootstrap.min.js"></script>
+    </head>
+    <body>
+        <h1>QA Settings</h1>
+        <p><b>{IF journal_name != ""}Journal: {journal_name}{ENDIF}</b></p>
+        <p>
+            <ul>
+                <li>always = MUST exist (an error will occur when trying to deliver without it)</li>
+                <li>sometimes = Already seen, may exist</li>
+                <li>ignore = No information about this tag yet</li>
+            </ul>
+        </p>
+        <table class="table table-bordered" style="width: 200px;">
+            <thead>
+                <tr>
+                    <th>Tag</th>
+                    <th>Global<br>(All&nbsp;journals)</th>
+                    <th>Journal<br>(This&nbsp;journal)</th>
+                </tr>
+            </thead>
+            <tbody>
+                {LOOP tags, global_settings, journal_settings}
+                    <tr>
+                        <td><a href="https://www.loc.gov/marc/bibliographic/bd{tags}.html" title="Link to external MARC21 documentation at loc.gov" target="_blank">{tags}</a></td>
+                        <td>{global_settings}{IF global_settings != ""}<a href="?action=show_qa&zeder_id={zeder_id}&zeder_instance={zeder_instance}&delete_tag={tags}&delete_type=global" title="Delete this rule" onclick="return confirm('Are you sure?')"><sup>x</sup></a>{ENDIF}</td>
+                        <td>{journal_settings}{IF journal_settings != ""}<a href="?action=show_qa&zeder_id={zeder_id}&zeder_instance={zeder_instance}&delete_tag={tags}&delete_type=local" title="Delete this rule" onclick="return confirm('Are you sure?')"><sup>x</sup></a>{ENDIF}</td>
+                    </tr>
+                {ENDLOOP}
+            </tbody>
+        </table>
+
+        <form id="add_form" method="post" action="?" autocomplete="off" onsubmit="return confirm('Are you sure?')">
+            <input type="hidden" name="action" value="show_qa">
+            <input type="hidden" name="zeder_id" value="{zeder_id}">
+            <input type="hidden" name="zeder_instance" value="{zeder_instance}">
+
+            <p>Add new setting:</p>
+            <!-- The quantifier inside the pattern needs to be written as "{{3}" due to Template.h escaping rules! -->
```

Comment:
This comment will end up in the generated HTML where it does not make any sense.  Do we need template comments that get not passed through to the generated HTML or whatever else we may be generating?

Reply:
Well that might come in handy in such a situation... because if you don't comment that, {{3} might look like a bug. But i think such a feature would only be nice-to-have